basculehttp/bearerTokenFactory_test.go

```diff
@@ -123,18 +123,18 @@ func TestBearerTokenFactory(t *testing.T) {
 	for _, tc := range tests {
 		t.Run(tc.description, func(t *testing.T) {
 			assert := assert.New(t)
-			r := new(key.MockResolver)
+			r := new(MockResolver)
 			p := new(mockParser)
-			pair := new(key.MockPair)
+			key := new(MockKey)
 			if tc.parseCalled {
 				token := jwt.NewWithClaims(jwt.SigningMethodHS256, tc.claims)
 				token.Valid = tc.validToken
 				p.On("ParseJWT", mock.Anything, mock.Anything, mock.Anything).Return(token, tc.parseErr).Once()
 			}
 			if tc.resolveCalled {
-				r.On("ResolveKey", mock.Anything, mock.Anything).Return(pair, tc.resolveErr).Once()
+				r.On("Resolve", mock.Anything, mock.Anything).Return(key, tc.resolveErr).Once()
 				if tc.resolveErr == nil {
-					pair.On("Public").Return(nil).Once()
+					key.On("Public").Return(nil).Once()
```

Comment:
You'll want to add `pair.AssertExpectations(t)` and `key.AssertExpectations(t)` to ensure mock calls were made

Reply:
yep, thanks for the reminder. I am still getting used to unit testing in Go. This has been updated in my most recent commit.